webpack/scenes/ContentViews/Copy/CopyContentViewForm.js

```diff
@@ -61,7 +61,7 @@ const CopyContentViewForm = ({ cvId, setModalOpen }) => {
         <Button
           aria-label="copy_content_view"
           variant="primary"
-          isDisabled={name.length <= 2 || saving}
+          isDisabled={name.length <= 1 || saving}
```

Comment:
I was going to raise an issue for this..:)

Reply:
Yeah, I actually went and updated it because I couldn't name one of my content views 'Io'... 😵‍💫

Reply:
We can still have 1 letter CV names..We can use name.length < 1

Reply:
kk